Jenkinsfile

```diff
@@ -18,22 +17,14 @@ slaveTemplates.dockerTemplate { label ->
       stage('Build image') {
         withCredentials([string(credentialsId: 'aws_account_id', variable: 'aws_account_id')]) {
           def awsRegistry = "${env.aws_account_id}.dkr.ecr.eu-central-1.amazonaws.com"
-          def imageVersion = "${env.BRANCH_NAME}".replaceAll('[-+/%_ ]','').toLowerCase()
-
-          sh "docker build \
-            -t ${awsRegistry}/${imageName}:${imageVersion} \
-            -t ${awsRegistry}/${imageName}:${scmVars.GIT_COMMIT} \
-            -t ${ghcrRegistry}/santiment/${imageName}:${imageVersion} \
-            -f docker/Dockerfile ."
-
           docker.withRegistry("https://${awsRegistry}", "ecr:eu-central-1:ecr-credentials") {
-            sh "docker push ${awsRegistry}/${imageName}:${imageVersion}"
+            sh "docker build \
```

Comment:
Not related with the main PR but we started getting Jenkins build errors. The issue is that we are using Tzanko's personal credentials to push the image to public Github image repository. I think we can stop doing this until / if there is some demand for this.